recipes/centos_vertx/cccp.yml

```diff
@@ -0,0 +1 @@
+job-id: vertx
```

Comment:
@benoitf that file is needed for the centOS container pipeline to build it. @dharmit As for PR https://github.com/eclipse/che/pull/2643#issuecomment-253166429 can you add a comment that explains what is this file for?